csrc/disjoint_set.h

```diff
@@ -153,25 +153,25 @@ class VectorOfUniqueEntries {
 
   // Returns first element in vector
   T front() const {
-#if defined(NDEBUG) && !defined(NVFUSER_EXPLICIT_CHECK)
+#if defined(NDEBUG) && !defined(NVFUSER_EXPLICIT_ERROR_CHECK)
```

Comment:
This still seems off to me. I think we want to do this assertion when debugging or the explicit check is required. So, shouldn't this be `not defined(NDEBUG) || defined(NVFUSER_EXPLICIT_ERROR_CHECK)`?

Reply:
good catch. You are totally right.
I think this was copied from the static_dynamic check earlier and I didn't notice it's originally calling `#ifndef` :face_holding_back_tears: 

https://github.com/NVIDIA/Fuser/commit/5a603dbf2ec01d503f41f93d1b9690eec70ee190

Reply:
Thanks for the fix. I really don't like `not defined(NDEBUG)` as the double negative is just confusing.